pkg/monitortests/authentication/nodefaultserviceaccountoperatorstests/monitortest.go

```diff
@@ -0,0 +1,247 @@
+package nodefaultserviceaccountoperatortests
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	configv1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/monitortestframework"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type noDefaultServiceAccountChecker struct {
+	kubeClient kubernetes.Interface
+	cfgClient  *configv1.ConfigV1Client
+}
+
+func NewAnalyzer() monitortestframework.MonitorTest {
+	return &noDefaultServiceAccountChecker{}
+}
+
+// Cleanup implements monitortestframework.MonitorTest.
+func (n *noDefaultServiceAccountChecker) Cleanup(ctx context.Context) error {
+	return nil
+}
+
+var exceptions = []func(pod corev1.Pod) bool{
+	func(pod corev1.Pod) bool {
+		podNameNSCombo := pod.Namespace + "/" + pod.Name
+		return strings.HasPrefix(podNameNSCombo, "openshift-cluster-version/cluster-version-operator-")
```

Comment:
Nit: Looks like this is a fairly repeatable pattern that we could turn into a common function for most of these exceptions?

Not a huge deal, but might save us a handful of lines of code.

Note, because this is a nit it isn't something I will block this PR from merging on and will leave it up to you to either take or leave the suggestion.